#include "centre_player_scene_replied_handler.h"
///<<< BEGIN WRITING YOUR CODE
///<<< END WRITING YOUR CODE
void CentrePlayerSceneServiceRepliedHandler::EnterScene(entt::entity player,const ::CentreEnterSceneRequest* request,
	     google::protobuf::Empty* response)
{
///<<< BEGIN WRITING YOUR CODE
///<<< END WRITING YOUR CODE
}

void CentrePlayerSceneServiceRepliedHandler::LeaveScene(entt::entity player,const ::CentreLeaveSceneRequest* request,
	     google::protobuf::Empty* response)
{
///<<< BEGIN WRITING YOUR CODE
///<<< END WRITING YOUR CODE
}

void CentrePlayerSceneServiceRepliedHandler::LeaveSceneAsyncSavePlayerComplete(entt::entity player,const ::CentreLeaveSceneAsyncSavePlayerCompleteRequest* request,
	     google::protobuf::Empty* response)
{
///<<< BEGIN WRITING YOUR CODE
///<<< END WRITING YOUR CODE
}

void CentrePlayerSceneServiceRepliedHandler::SceneInfoC2S(entt::entity player,const ::SceneInfoRequest* request,
	     google::protobuf::Empty* response)
{
///<<< BEGIN WRITING YOUR CODE
///<<< END WRITING YOUR CODE
}

